src/components /FormMood.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import {
+  BsEmojiSmile,
+  BsEmojiExpressionless,
+  BsEmojiFrown,
+} from "react-icons/bs";
+
+const FormMood = (props) => {
```

Comment:
I see some replication across these three forms. Maybe you could define a prop to be used, to determine how many inputs to render, and what the attributes would be. 

```suggestion
const moodInputs = [{type: "date", id: "mood-date", required: true, placeholder: "", ...}]
```

or something else along these lines. You could define any number of props on <FormMood /> by the way and use them here in this component via ...props
<input ...props /> for example.

Another idea would be to create an input component, and pass props as needed to it, for the dynamic props. 
<Input value={props.moodDate} onChange={props.onDateChange} />. This component would include the styling already, not necessary to define here then.

Reply:
Overall we would want to try to reduce the replication here by all means.